core/src/functions/moveElementInArray/moveElementInArray.ts

```diff
@@ -0,0 +1,16 @@
+export const moveElementInArray = <T>(
+  entries: Array<T>,
+  moveFromIndex: number,
+  moveToIndex: number,
+): Array<T> => {
+  if (moveFromIndex < 0 || moveFromIndex >= entries.length) {
+    console.error("moveFromIndex out of bounds");
```

Comment:
I would try to avoid console.errors and instead clamp the values..
e.g.

```
const clampedIndex = Math.max(0, Math.min(entries.length, index));
```

Reply:
Nope.  In this case, this is not possible. For example, if you have an array with a length of 4 and want to move the 6 element, you cannot automatically take the 4, as the values are then incorrect.